src/core/EdFi.DataManagementService.Core/EdFi.DataManagementService.Core.csproj

```diff
@@ -19,6 +20,7 @@
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="8.0.1" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="8.0.1" />
+    <PackageReference Include="Npgsql" Version="8.0.3" />
```

Comment:
Are Npgsql and Dapper really required in the Core project? I would have thought they would be isolated to the backend projects.

Reply:
@stephenfuqua that's probably the end goal. This was to avoid more file shuffling and get started using the `backend` folder in core instead of the `backend` project.

Reply:
So long as we have a plan, temporary is fine.